src/core/loader/cia.cpp

```diff
@@ -146,13 +146,13 @@ FileType AppLoader_CIA::IdentifyType(FileUtil::IOFile& file) {
 
     return FileType::Error;
 }
-ResultStatus AppLoader_CIA::ReadProgramId(u64& out_program_id) {
-    cia_context ctx;
-    ReadCiaFile(ctx);
-    auto cool = ctx.tik_feild.tik.title_id;
-    std::array<u8, 8> mojojojo;
-    std::copy(std::begin(cool), std::end(cool), std::begin(mojojojo));
-    out_program_id = *mojojojo.data();
-    return ResultStatus::Success;
-}
+//ResultStatus AppLoader_CIA::ReadProgramId(u64& out_program_id) {
+//    cia_context ctx;
+//    ReadCiaFile(ctx);
+//    auto cool = ctx.tik_feild.tik.title_id;
+//    std::array<u8, 8> mojojojo;
+//    std::copy(std::begin(cool), std::end(cool), std::begin(mojojojo));
+//    out_program_id = *mojojojo.data();
+//    return ResultStatus::Success;
+//}
```

Comment:
even if it's just temporarily, use

```
#if 0
code
#endif
```

to comment out code.
Reserve `//` and `/* comments */` for actual descriptions / notes to humans